conf/config-default.yaml

```diff
@@ -115,6 +115,8 @@ nginx_config:                     # config for render the template to genarate n
   worker_shutdown_timeout: 240s     # timeout for a graceful shutdown of worker processes
   event:
     worker_connections: 10620
+  envs:
+    - APISIX_PROFILE
```

Comment:
Since default configure can be overridden by the user, I think we should always include `APISIX_PROFILE` in the nginx.conf template. Otherwise, `APISIX_PROFILE` might be lost by mistake. We can leave a comment here as example.

Reply:
From the definition of APISIX_PROFILE, you are right. But I think this configuration should be defined in the configuration file instead of environment variables. I think can use the environment variable definition part of the configuration. When there is a corresponding environment variable definition, the environment variable is preferred

Reply:
the above are my thoughts. i modifyd a version first will not cover `APISIX_PROFILE`